src/test/scala/CourseAccessTests.scala

```diff
@@ -1,85 +1,67 @@
+import de.upb.cs.uc4.hyperledger.connections.traits.ConnectionCourseTrait
+import de.upb.cs.uc4.hyperledger.exceptions.TransactionException
+import de.upb.cs.uc4.hyperledger.testBase.TestBaseDevNetwork
 
-import java.nio.file.Paths
+class CourseAccessTests extends TestBaseDevNetwork {
 
-import de.upb.cs.uc4.hyperledger.ConnectionManager
-import de.upb.cs.uc4.hyperledger.traits.ChaincodeActionsTrait
-import org.scalatest.Succeeded
-import org.scalatest.matchers.should.Matchers
-import org.scalatest.wordspec.AnyWordSpec
+  var chaincodeConnection: ConnectionCourseTrait = _
 
-import scala.util.{Success, Using}
-
-class CourseAccessTests extends AnyWordSpec with Matchers {
+  override def beforeEach() {
```

Comment:
We are using this style:
```scala
override def beforeEach() = {
```

Reply:
jop, hätte m.M.n. sogar der compiler anmeckern sollen